package.json

```diff
@@ -409,6 +409,13 @@
                   "type": "string"
                 },
                 "default": []
+              },
+              "env": {
+                "type": "object",
+                "items": {
+                  "type": "string"
+                },
+                "default": []
```

Comment:
Should the default still be `[]` or `{}`?

Reply:
Fixed!